GNUmakefile

```diff
@@ -22,3 +22,13 @@ help:
 .PHONY: test
 test: ## Run all tests
 	$Q $(CURDIR)/test.sh --type oss --unprivileged false --latest-njs false
+
+out:
```

Comment:
Two comments:
1. You can probably use `npx jsdoc -c` rather than directly referencing the js file which could change out from under us.
2. `out` seems like a rather generic name for building the docs.  `docs` might be more intuitive?